.github/actions/update-pom-to-next-pre-release/action.yml

```diff
@@ -0,0 +1,28 @@
+description: Update pom files to the next pre-release
+name: Update pom to next pre-release
+outputs:
+  next-prerelease:
+    description: "Next prerelease"
+    value: ${{ steps.next-prerelease-resolver.outputs.next-prerelease }}
+runs:
+  using: composite
+  steps:
+    - name: Parse next version from POM
+      id: parse-next-final-version
+      shell: bash
+      run: |
+        NEXT_VERSION=$(yq -p=xml e '.project.version' pom.xml | grep -o "[0-9]*\.[0-9]*.[0-9]*")
+        echo "Next final version: $NEXT_VERSION"
+        echo "::set-output name=result::$NEXT_VERSION"
+
+    - id: next-prerelease-resolver
+      name: Calculate next internal release
+      uses: Alfresco/alfresco-build-tools/.github/actions/calculate-next-internal-version@v1.5.0
```

Comment:
My understanding (reading the doc) is that you can reference this local action using instead:
```
uses: ./.github/actions/calculate-next-internal-version/action.yml
```
This would allow not having to mention the version (and just use the current one), but I have not tested that it works ok.

Reply:
Also version will be v1.6.0, you will need to tag the repo with this version when the PR has been merged.

Reply:
Also version will be v1.6.0, you will need to tag the repo with this version when the PR has been merged.

Reply:
that's actually the most confusing thing about reusable actions/workflow: referencing an action via a relative/local path would work *only* in this repository. When this action will be used on any other repository that local path don't exists.

Last time we discussed about this I remember that is something that probably GitHub has to solve on their side because with the current status we are pretty limited to use global reference (Alfresco/alfresco-build-tools) and so an explicit ref as version

Reply:
Indeed Elias explained to me that this does not work as I was expecting :(

Reply:
AFAIK, the only way to have reusable actions / workflows working properly doesn't matter from where they are called, is by referencing them using the repo and version. Unfortunately, we cannot use context expressions to reference the version either.